.codecov.yml

```diff
@@ -2,9 +2,9 @@ codecov:
   require_ci_to_pass: no
   notify:
     # after_n_builds calculation
-    # 10 = 1 (lin-py36) + 4 (win-py37) + 4 (func-s_{base,tf115,tf21,tf25}-lin-py37) + 1 (pylaunch)
+    # 11 = 1 (lin-py36) + 5 (win-py37) + 4 (func-s_{base,tf115,tf21,tf25}-lin-py37) + 1 (pylaunch)
```

Comment:
how is the windows count 5?  parallelism is set to 4 for windows.. it must be something else?

Reply:
maybe an issue when flaky test fail, and that counts as a run so you have to have a clean set of 10 workflows run.